bin/mysql-schema-server-petstore.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Minor suggestion: mysql-schema-server-petstore.sh => mysql-schema-petstore.sh

(don't want people to get confused it's a server stub generator)